boards/jiminy-mega256rfr2/Makefile.include

```diff
@@ -0,0 +1,26 @@
+# define the cpu used by the jiminy board
+export CPU = atmega256rfr2
+
+USEMODLUE += periph
```

Comment:
typo on USEMODULE

Reply:
by the way this line is also present in the CPU Makefile.include. is it really needed here ?